src/scripts/components/app/app.tsx

```diff
@@ -36,6 +38,19 @@ const App: FunctionComponent = () => (
 
 const TranslatedApp: FunctionComponent = () => {
   const language = useSelector(languageSelector);
+  const selectedLayers = useSelector(selectedLayerIdsSelector);
```

Comment:
we could put this stuff in a custom hook 'const markers = useStoryMarkers()' so that we keep the `app.tsx` clean

Reply:
also only map over the stories if hideMarkers is false to save some work if we don't show the markers